docs/migrate/rockset/index.md

```diff
@@ -0,0 +1,365 @@
+show that CrateDB Cloud is in a similar ballpark and oftentimes a bit cheaper
+than solutions from other vendors.
+:::::
+
+
+
+:::{rubric} Benefits
+:::
+CrateDB's benefits and principles in a nutshell.
+
+:::::{grid} 1 1 2 2
+:gutter: 3
+
+::::{grid-item-card}
+:::{rubric} Fundamentals
+:::
+CrateDB is a scalable and cost-effective real-time analytics database,
+combining complex JSON handling, time series, geospatial data, full-text
+search, and vector search in one single storage engine.
+
+- No vendor lock-in and service sunsetting woes.
+- No capacity limits.
+
+Built on top of Apache Lucene and Elasticsearch, CrateDB automatically
+indexes all your data to achieve millisecond response times for any kind
+of incoming query and aggregation.
+
+- [All features of CrateDB at a glance]
+- [The CrateDB Documentation]
+
+CrateDB is offered as a managed service available on AWS, Azure, and GCP,
+and also as a fully open source edition.
+- [CrateDB Cloud]
+- [CrateDB Editions]
+::::
+
+::::{grid-item-card}
+:::{rubric} Interoperability, Open Source, Open Standards
+:::
+Rockset is a proprietary solution available as a managed service in AWS only.
+
+With CrateDB, you leverage open standards to avoid vendor lock-in and
+future-proof your systems.
+
+CrateDB's [open-source code base] accepts contributions from community
+members and corporates, so you can easily add features you need, and
+distribute them to downstream users in multiple ways, without any
+licensing hassles.
+
+:::{rubric} Development and Embedding
+:::
+The attributes enumerated above allow you to integrate CrateDB into your
+products seamlessly, and supply your engineers and continuous integration
+systems with dedicated instances of CrateDB, instead of needing to run
+them against a cloud-only service, which is slowing down development,
+mostly due to API rate-limiting measures.
+::::
+
+:::::
+
+:::{rubric} Features
+:::
+CrateDB's functional features and integration capabilities in a nutshell.
+
+:::::{grid} 1 1 2 2
+:gutter: 3
+
+::::{grid-item-card}
+:::{rubric} SQL Query Language
+:::
+CrateDB's lingua franca is SQL, ready for big data, very similar to
+Rockset's SQL dialect.
+
+Because CrateDB is also compatible with applications from the PostgreSQL
+ecosystem, you can leverage many of them without further ado.
+
+- [Advanced Querying]
+:::{todo}
+Link to SQL function overview/comparison shared by @hlcianfagna on another PR.
+:::
+::::
+
+::::{grid-item-card}
+:::{rubric} Integrations
+:::
+With Rockset, as a complete data warehouse system, integrating data from other
+sources is a nobrainer, and provides exceptional UX/DX.
+
+CrateDB offers a wide range of integration capabilities, is compatible
+with the PostgreSQL wire protocol, and offers adapter components to
+integrate with applications and frameworks.
+
+- [Ecosystem Catalog]
+- [Integration Tutorials I]
+- [Integration Tutorials II]
+- [Software Development Kit]
```

Comment:
Yeah, we still have those two entrypoints to tutorials resources. It needs to be consolidated, but it is not a concern of the current iteration.

Reply:
Diverted to GH-104.